UefiPayloadPkg/UefiPayloadEntry/BuildHobsUniversal.c

```diff
@@ -18,5 +107,75 @@ BuildHobs (
   VOID
   )
 {
+  EFI_PEI_HOB_POINTERS             Hob;
+  UINTN                            MinimalNeededSize;
+  UINTN                            FreeMemoryBottom;

```

Comment:
Since the data type in PHIT HOB is EFI_PHYSICAL_ADDRESS, can we align that to not use UINTN?